README.md

```diff
@@ -1,20 +1,88 @@
 # Flow Data Provisioning Service
 
+The Flow Data Provisioning Service aims at providing a scalable and efficient way to access the execution state of past
+and live sporks, as well as implementing the Rosetta API standard.
+
+Past sporks are loaded from the filesystem using Flow's Write-Ahead Log to replay the changes to all registries, and
+updates are pushed directly to the DPS by the execution nodes in order to handle live networks.
+
+The Flow DPS maintains different specialized indexes for different purposes. One index is used for accessing the entire
+execution state at any given height, while the other is used to follow the history of a specific registry through time.
+Contrary to Flow's WAL, those indexes allow replaying changes both backward and forward, which enables visualization
```

Comment:
`Contrary to the execution node's LedgerWAL, the indexes allow random access to the state trie at any block height, which enables state retrieval at any point in history and far beyond the execution node's pruning limit.`